endpoints.go

```diff
@@ -10,8 +10,9 @@ import (
 	"sort"
 	"strings"
 
-	cmds "github.com/ipfs/go-ipfs/commands"
+	cmds "gx/ipfs/QmfAkMSt9Fwzk48QDJecPcwCUjnf2uG7MLnmCGTp4C6ouL/go-ipfs-cmds"
```

Comment:
We only leave gx paths in this rewritten state in go-ipfs so that users don't build go-ipfs with `go get ...` and then complain when everything is broken. In other projects, we use the standard github.com/... paths.

Reply:
As it uses go-ipfs code it might make sense. Not sure though.

Reply:
Yeah. As it uses go-ipfs code, the compiler thinks the types are different if we import using standard github.com/... paths.

Reply:
Got it. Makes sense.